airflow/executors/local_executor.py

```diff
@@ -65,14 +74,31 @@ def execute_work(self, key: TaskInstanceKey, command: CommandType) -> None:
         if key is None:
             return
         self.log.info("%s running %s", self.__class__.__name__, command)
+        old_title = getproctitle()
         try:
-            subprocess.check_call(command, close_fds=True)
+            from airflow.cli.cli_parser import get_parser
+            parser = get_parser()
+            # [1:] - remove "airflow" from the start of the command
+            args = parser.parse_args(command[1:])
```

Comment:
I think this may raise similar concerns as #6905: that users will have to restart Airflow scheduler to reload plugins, right?

Reply:
Oh I hadn't seen that PR (or forgotten about it).

Hmm, not sure - yes plugins would be pre-loaded. I guess this Congress down to not recommending putting operators in plugins too.

I'll add a config flag (on by default?) to go back to the old way

Reply:
DAGs in plugins are one thing, custom operators are other and I think many users keep custom operators as plugins

Reply:
My point is that custom operators don't need to be in plugins - just a normal python module, and then they won't be loaded by these long running process. See #9498 and #9500.

This concern still applies to hooks though.

Reply:
Done.